src/ros_filter.cpp

```diff
@@ -771,21 +771,20 @@ namespace RobotLocalization
       else
       {
         ROS_WARN_STREAM("use_control is set to true, but acceleration_limits is missing. Will use default values.");
-        accelerationLimits.resize(TWIST_SIZE, 1.0);
```

Comment:
This was doing nothing, so the default was all `0.0`. If the intention is to have all `0.0`, that can be changed on the construction before. The `resize` does nothing here. Same for the `resize` below to the gains.

I've change the default from `0.0` to `1.0` because if the limits are `0.0` by default, then the gain defaulting to `1.0` won't do anything. IMHO it's more intuitive/expectable to have the limits default to `1.0`.